Makefile

```diff
@@ -13,6 +13,8 @@ OPEN_RESOURCE=bash -c 'open $$0 || xdg-open $$0'
 SLIDESHOW_TO_PDF?=bash -c 'docker run --rm -v `pwd`:/cwd astefanutti/decktape /cwd/$$0 /cwd/`dirname $$0`/`basename -s .html $$0`.pdf'
 YARN := $(shell command -v yarn 2> /dev/null)
 YARN_INSTALL_OPTS=--network-timeout 300000 --check-files
+NODE_OPTIONS ?= --max-old-space-size=3072
```

Comment:
Would you mind adding a comment with explanation here? I will look at this in 6 (likely 2) months and will have no idea what is happening.

Reply:
Sure thing, done!

Reply:
thank you!